accelerate/remix-starter/app/components/Quote/Quote.tsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable react/prop-types */
+import { QuoteWrapper } from "./QuoteWrapper";
+import { QuoteCacheType, QuoteResult } from "./../../lib/types";
+import pkg from "openflights-cached";
+const { findIATA } = pkg;
+
+export const Quote: React.FC<{
+  title: string;
+  type: QuoteCacheType;
+  result: QuoteResult;
+}> = ({ title, type, result }) => {
+  const [{ id, quote, createdAt }, { cacheStatus, region, lastModified }] = [
+    result.data,
+    result.info,
+  ];
+
+  return (
+    <QuoteWrapper title={title} type={type}>
+      <div className="flex flex-col">
+        <p className="text-lg">
+          {" "}
```

Comment:
I see we have this in some `p` tags and not others. Any reason we need this whitespace?

Reply:
I added whitespace through CSS. Good catch.